aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -85,6 +94,11 @@ module AptosFramework::Coin {
         borrow_global<CoinInfo<CoinType>>(coin_address).supply
     }
 
+    /// Returns the `value` of the passed in `coin`. 
+    public fun value<CoinType>(coin: &Coin<CoinType>): u64 {
```

Comment:
this should be in public functions for the coin, which is below -- these are public functions for the CoinType

Reply:
Moved in recent commit